projects/Mallard/src/helpers/files.ts

```diff
@@ -172,16 +172,21 @@ export const fetchAndStoreIssueSummary = async (): Promise<IssueSummary[]> => {
     const fetchIssueSummaryUrl = `${apiUrl}${edition}/issues`
 
     try {
-        await RNFetchBlob.config({
-            overwrite: true,
-            path: FSPaths.contentPrefixDir + defaultSettings.issuesPath,
-            IOSBackgroundTask: true,
-        }).fetch('GET', fetchIssueSummaryUrl, {
-            'Content-Type': 'application/json',
-        })
+        const issueSummaryRequest = await fetch(fetchIssueSummaryUrl)
+        const issueSummary = await issueSummaryRequest.json()
+        if (!issueSummary) {
+            throw new Error('No Issume Summary Avaialble')
+        }
+
+        const issueSummaryString = JSON.stringify(issueSummary)
```

Comment:
Do we need to stringfy to write to fs?

Reply:
We do on the basis that we parse when we get back in another function but that could be refactored to remove both.